src/app/view/components/_common/dot-empty-state/dot-empty-state.component.scss

```
@@ -0,0 +1,97 @@
+@use "variables" as *;
+
+:host {
+    position: relative;
+    display: block;
+
+    .dot-empty-container__notice {
+        .material-icons {
+            color: $brand-primary;
+            font-size: 4rem;
+        }
+        position: absolute;
+        padding: $spacing-3 $spacing-5;
+        top: 50%;
+        left: 50%;
+        z-index: 1;
+        transform: translateY(-50%) translateX(-50%);
+        display: flex;
+        flex-direction: column;
+        align-items: center;
```

Best read this way..

```suggestion
        position: absolute;
        padding: $spacing-3 $spacing-5;
        top: 50%;
        left: 50%;
        z-index: 1;
        transform: translateY(-50%) translateX(-50%);
        display: flex;
        flex-direction: column;
        align-items: center;
        
        .material-icons {
            color: $brand-primary;
            font-size: 4rem;
        }
```